src/NuGet.Clients/NuGet.PackageManagement.UI/Xamls/PackageManagerControl.xaml.cs

```diff
@@ -1458,7 +1459,7 @@ private void PackageList_UpdateButtonClicked(PackageItemViewModel[] selectedPack
                 .Select(package => new PackageIdentity(package.Id, package.Version))
```

Comment:
I think we should remove this code because `packagesToUpdate` variable is no longer used while calling `UpdatePackages` method.

Reply:
After all, I will use this code.